roles/manage-users/tests/vars/idm_mod.json

```diff
@@ -1,39 +1,25 @@
 {
     "users" : [
-        { 
+        {
             "user_name": "gdownie",
             "first_name": "Gord",
-            "last_name": "Downie",
+            "last_name": "Donie",
```

Comment:
I know you're trying to test a "change" here, but it will leave the reader with the impression that this is a typo ...